scripts/build-tools.js

```diff
@@ -1,24 +1,33 @@
 const { getData } = require('./tools/extract-tools-github');
 const { convertTools } = require('./tools/tools-object');
 const { combineTools } = require('./tools/combine-tools');
-const manualTools = require('../config/tools-manual.json')
-
 const fs = require('fs');
 const { resolve } = require('path');
 
-const buildTools = async () => {
+const buildTools = async (automatedToolsPath, manualToolsPath, toolsPath, tagsPath) => {
   try {
     let githubExtractData = await getData();
     let automatedTools = await convertTools(githubExtractData);
+    
     fs.writeFileSync(
-      resolve(__dirname, '../config', 'tools-automated.json'),
+      automatedToolsPath,
       JSON.stringify(automatedTools, null, '  ')
     );
-    await combineTools(automatedTools, manualTools);
+    
+    await combineTools(automatedTools, require(manualToolsPath), toolsPath, tagsPath);
```

Comment:
This function doesn't even accept more than 2 parameters, any reason behind this addition of toolsPath and tagsPath?